physlr/physlr.py

```diff
@@ -1430,21 +1430,36 @@ def physlr_count_molecules(self):
     @staticmethod
     def split_minimizers_bx(bx, g, bxtomxs):
         "Partition the minimizers of the given barcode"
-        bx_match = re.compile(r'^(\S+)_\d+$')
+        bx_match = re.compile(r'^(\S+)_\d+_\d+$')
         mxs = bxtomxs[bx]
         mol = 0
+        mol2 = 0
         mol_list = []
-        while g.has_node(bx + "_" + str(mol)):
-            bxmol = bx + "_" + str(mol)
-            neighbour_mxs_list = [bxtomxs[re.search(bx_match, v).group(1)] \
-                                  for v in g.neighbors(bxmol) \
-                                  if re.search(bx_match, v).group(1) in bxtomxs]
-            if not neighbour_mxs_list:
-                neighbour_mxs_list = [set()]
-            neighbour_mxs_set = set.union(*neighbour_mxs_list)
-            molec_mxs = set.intersection(mxs, neighbour_mxs_set)
-            mol_list.append((bxmol, molec_mxs))
+        while g.has_node(bx + "_" + str(mol)+ "_" + str(mol2)):
+            while g.has_node(bx + "_" + str(mol)+ "_" + str(mol2)):
+                bxmol = bx + "_" + str(mol)+ "_" + str(mol2)
+                neighbour_mxs_list = [bxtomxs[re.search(bx_match, v).group(1)] \
+                                      for v in g.neighbors(bxmol) \
+                                      if re.search(bx_match, v).group(1) in bxtomxs]
+                if not neighbour_mxs_list:
+                    neighbour_mxs_list = [set()]
+                neighbour_mxs_set = set.union(*neighbour_mxs_list)
+                molec_mxs = set.intersection(mxs, neighbour_mxs_set)
+                mol_list.append((bxmol, molec_mxs))
+                mol2 += 1
+                if not g.has_node(bx + "_" + str(mol)+ "_" + str(mol2)):
+                    for i in range(1,11):
```

Comment:
```suggestion
                    for i in range(1, 11):
```
pylint is getting mad because of this whitespace

Reply:
Also, why is this range hard-coded? Is it possible that there will be a `A_0_0` and `A_0_2` but not `A_0_1` for example?
If so, a more general solution could be to count the number of molecules per barcode (or molecules per molecule-split barcode) -- we have the function `count_molecules_per_bx`, and use that for the logic of the while loops.

Reply:
Yes to your first question. I'll have to look into that function. I hard-coded because I couldn't think of a simple way to deal with the problem and I figure 10 seems large enough to get it to work (which it did)